data/fcitx5-diagnose.sh

```diff
@@ -745,7 +732,7 @@ fcitx_lib_path=()
 init_fcitx_lib_path() {
     local path
     local __fcitx_lib_path
-    fcitx_lib_path=(/usr/lib/fcitx5/ /usr/local/lib/fcitx5/)
+    fcitx_lib_path=(/usr/lib/fcitx5/ /usr/local/lib/fcitx5/ @FCITX_INSTALL_ADDONDIR@)
```

Comment:
Single quote around this and use ESCAPE_QUOTES https://cmake.org/cmake/help/latest/command/configure_file.html